src/utils.ts

```diff
@@ -11,3 +15,69 @@ export function getFirefoxVersion() {
 
     return parseInt(match[1], 10);
 }
+
+/**
+ * Checks if a version is a development build
+ * @param version The version to check
+ * @returns true if the version is a development build, false otherwise
+ */
+export function isDevBuild(version: string | undefined): boolean {
+    if (!version) {
+        return false;
+    }
+
+    // Consider "master" as a dev build
+    if (version === 'master') {
+        return true;
+    }
+
+    // Check if version has "dev" as a prefix.
+    return version.startsWith('dev');
+}
+
+/**
+ * Checks if a version meets or exceeds a minimum version requirement
+ * @param version The version to check
+ * @param minVersion The minimum version required
+ * @returns true if version >= minVersion or if it's a dev build, false otherwise
+ */
+export function isVersionAtLeast(version: string | undefined, minVersion: string): boolean {
+    if (!version) {
+        return false;
+    }
+
+    // Dev builds are considered to have the latest features
+    if (isDevBuild(version)) {
+        return true;
+    }
+
+    try {
+        return semver.gte(version, minVersion);
```

Comment:
does this do the `1.7.0-rc1` < `1.7.0` thing? Would we ever use rc builds?

Reply:
Good question. I think so, but since we support dev builds separately, I think it may be fine. I'll make it explicit though, through a test case.